src/common/output/compare.rs

```diff
@@ -0,0 +1,28 @@
+use crate::common::{CompareWarning, FileEntry};
+
+pub struct CompareOutput {
+    // Quiet program output mode
+    is_quiet_mode: bool,
+}
+
+impl CompareOutput {
+    pub fn new(is_quiet_mode: bool) -> Self {
+        CompareOutput { is_quiet_mode }
+    }
+
+    /// Prints information about a file in process
+    pub fn print_processing_info(&self, file: &FileEntry) {
+        if !self.is_quiet_mode {
+            println!("Collecting keys of file {}", file);
+        }
+    }
+
+    /// Prints warnings without any additional information
+    pub fn print_warnings(&self, warnings: &Vec<CompareWarning>) {
+        warnings.iter().for_each(|w| println!("{}", w.as_str()));
+
+        if self.is_quiet_mode {
```

Comment:
I think this is an extra check here, because output occurs before 🙂

Reply:
Thanks, just removed it :)